Assets/Scripts/Models/Furniture.cs

```diff
@@ -20,7 +20,7 @@
 /// InstalledObjects are things like walls, doors, and furniture (e.g. a sofa).
 /// </summary>
 [MoonSharpUserData]
-public class Furniture : IXmlSerializable, ISelectable, IContextActionProvider
+public partial class Furniture : IXmlSerializable, ISelectable, IContextActionProvider
```

Comment:
hem I'm not a fan either of partial, if you have to partial a class you can probably split it in two for separation of concerns.